helix-core/src/main/java/org/apache/helix/HelixCloudProperty.java

```diff
@@ -141,10 +160,19 @@ public List<String> getCloudInfoSources() {
     return _cloudInfoSources;
   }
 
+  @Deprecated
```

Comment:
Let' not deprecate it. Since we decide move forward with pkg name + processor name. Then it would be valuable to let user get process name and pkg name separately.